### PR TITLE
Unit testing added

### PR DESCRIPTION
**Pull request by akshita27**

### CODE REVIEW

File: url_shortener/tests.py [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+                long_url="http://www.linkedin.com"
+        )
+        data = {'client_response': "http://www.twitter.com"}
+
+        resp = self.client.post('/home/shorturl/', data, format='json')
+        self.assertEqual(resp.status_code, 200)
+
+        data = {'client_response': ""}
+        resp = self.client.post('/home/shorturl/', data, format='json')
+        self.assertEqual(resp.status_code, 404)
```
Comment (vishesh92) [inline url_shortener/tests.py]: I don't think the status code should be 404.
Check for the returned URL as well.

File: url_shortener/tests.py [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+        self.assertEqual(resp.status_code, 405)
+
+        url_shortener_1 = MapUrls.objects.create(
+                id=101,
+                short_url="bN",
+                long_url="http://www.stackoverflow.com"
+        )
+
+        resp = self.client.get('/home/shorturl/bN')
+        self.assertEqual(resp.status_code, 200)
```
Comment (vishesh92) [inline url_shortener/tests.py]: Check for returned URL.

File: url_shortener/tests.py (hunk 1 above)
Comment (dash1291) [inline url_shortener/tests.py]: does this test case make much sense?

File: url_shortener/tests.py [hunk 3]
```diff
@@ -1,3 +1,41 @@
...
 
-# Create your tests here.
+
+class UrlShortenerViewsTestCase(TestCase):
+    def test_get_shorturl(self):
+        resp = self.client.get('/home/shorturl/')
+        self.assertEqual(resp.status_code, 405)
+
+        url_shortener_1 = MapUrls.objects.create(
+                id=101,
```
Comment (dash1291) [inline url_shortener/tests.py]: too much spacing here 👎

Comment (akshita27) [inline url_shortener/tests.py]: fixed...

**Pull request merged**